kfs-cam/src/main/resources/edu/arizona/kfs/module/cam/spring-cam.xml

```diff
@@ -21,6 +21,12 @@
         <property name="dataDictionaryPackages">
             <list merge="true">
                 <value>classpath:edu/arizona/kfs/module/cam/document/datadictionary/*.xml</value>
+                <value>classpath:edu/arizona/kfs/module/cam/businessobject/datadictionary/*.xml</value>
+            </list>
+        </property>
+        <property name="databaseRepositoryFilePaths">

```

Comment:
I don't see why this OJB table override is needed when the new edu.arizona BO class extends org.kuali.kfs.module.cam.businessobject.AssetRetirementGlobal and there are no changed or new database fields or relationships included in this modification.